src/playlist/Playlist.hpp

```diff
@@ -166,7 +166,7 @@ class Playlist
      * @param predicate The sort predicate.
      * @param order The sort order.
      */
-    virtual void Sort(uint32_t startIndex, uint32_t count, SortPredicate predicate, SortOrder order);
+    virtual void Sort(size_t startIndex, size_t count, SortPredicate predicate, SortOrder order);
```

Comment:
Even if the type would be better for the underlying containers, playlists won't ever exceed 4.2 billion presets. Better convert the types in the implementation.

Reply:
Agree, none of the instances of size_t should really matter for this application. size_t made more sense to me since it's an index, but I'm fine with casting especially in this case.